docs/components/home/Homepage.vue

```diff
@@ -0,0 +1,53 @@
+<script setup lang="ts">
+</script>
+
+<template>
+  <main class="main">
+    <article class="container border-red">
+      <div class="intro">
+        <p>
+          The documentation concerning
+          <br>
+          the npm modules of
+        </p>
+        <h1 class="title">
+          Shiki-Twoslash
+        </h1>
+        <p>
+          In which markdown code samples are powered by
+          <br>
+          the syntax engine of Visual Studio Code
+          <br>
+          mixed with the TypeScript compiler’s information
+        </p>
+
+        <div style="margin-top: 3rem; margin-bottom:3rem">
+          <img style="margin: 0 auto" src="/svgs/squiggle.svg" alt="Decoration" width="70" height="25.5">
+        </div>
+
+        <div class="intro">
+          <p class="by">
+            By <a href="https://orta.io">orta therox</a>
+          </p>
+          <p>
+            Purveyor of <a href="https://puzzmo.com/">Puzzles</a>, shipper of renowned open source code
+            <br>
+            and ex-TypeScript compiler team member
+          </p>
+
+          <p class="by" style="margin-top: 1rem">
+            And <a href="https://orta.io">Anthony Fu</a>
+          </p>
+
+          <p>
+            A veritable open sourceror.
+          </p>
+        </div>
+
+        <div style="margin-top: 3rem">
+          <img style="margin: 0 auto" src="/svgs/logo.svg" alt="Shiki Logo" width="167" height="238">
```

Comment:
```suggestion
          <img style="margin: 0 auto" src="/logo.svg" alt="Shiki Logo" width="167" height="238">
```

For the logo, I made a modified version by adding a beige background so it works better on dark backgrounds. Not sure what you would prefer:

<img width="735" alt="image" src="https://github.com/twoslashes/twoslash/assets/11247099/51070673-a9b1-4907-8b30-fa9cae907910">

Source is here btw: https://www.figma.com/file/sCQBSGPqSASpiql84BZ8Ct/TwoSlash?type=design&node-id=0%3A1&mode=design&t=CaY0IU8Xjz9NIBvo-1